src/main/kotlin/com/example/estdelivery/adapters/in/web/member/dto/request/MemberCommand.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.estdelivery.adapters.`in`.web.member.dto.request
+
+import com.example.estdelivery.domain.member.Member
+
+data class MemberCommand(
+    val name: String = ""
+)
+
+fun MemberCommand.toCreateDomain() = Member(name = name)
```

Comment:
동일하게 toDomain이면 좋아보입니다 ㅎㅎ

```suggestion
fun MemberCommand.toDomain() = Member(name = name)
```